pkg/apis/serving/v1alpha1/revision_lifecycle.go

```diff
@@ -52,11 +52,11 @@ const (
 
 	// UserQueueMetricsPortName specifies the port name to use for metrics
 	// emitted by queue-proxy for end user.
-	UserQueueMetricsPortName = "user-metrics"
+	UserQueueMetricsPortName = "http-usermetric"
```

Comment:
@mattmoor is this acceptable from an update PoV? I feel like, because these names are used when generating the deployment, we effectively trigger a redeployment when this change is applied after an update.

Reply:
We should be able to safely rollout changes to the user Deployment

Reply:
cc @tcnghia

Reply:
Ack.  I think this may be ok.  Following the conventions may help us avoid strange errors.